ethereum/swap_creator_test.go

```diff
@@ -409,7 +414,7 @@ func testRefundAfterT1(t *testing.T, asset ethcommon.Address, erc20Contract *Tes
 	}
 
 	nonce := big.NewInt(0)
-	timeout := big.NewInt(1) // T1 expires before we get the receipt for new_swap TX
+	timeout := big.NewInt(3) // T1 expires before we get the receipt for new_swap TX
```

Comment:
Is that comment still true after increasing the timeout to 3 seconds?

Reply:
no, can remove